### ISSUE TITLE
[Bug] Ctrl-R doesn't work

### ISSUE THREAD

**Issue by BellLongworth** [Zettlr/Zettlr#2311]
### Description

Expect a prompt with ability to enter new name. Instead nothing happens.

### Reproducing

Do ctrl-r

### Zettlr Version

Beta (if applicable)

### Specify version

beta-1

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 10

**Comment by nathanlesage** [comment]
> If the current file is not currently visible in the file list (out of the scroll area) then nothing visible happens.

Good point, I should add another scrollspy so that the list scrolls to that file if applicable.

> If the current file is visible, the change when pressing Ctrl-R is barely noticeable. A slightly different shade of blue and the ".md" extension appears.

This sounds as if it can be resolved with some additional styling. Would you mind making a screenshot for better visual cues?

**Comment by BellLongworth** [comment]
![image](https://user-images.githubusercontent.com/63842998/129485288-a8a1cfda-6338-4beb-8e86-7960e8cff75d.png)
Top is when the fie is selected. Bottom is after ctrl-r.
